packages/design-system/src/components/slider/slider.css.ts

```diff
@@ -148,6 +164,8 @@ export const sliderClassNames: SliderClassNames = {
       height: sliderSpaceVars.track.height,
       margin: sliderSpaceVars.track.margin,
       right: 'auto',
+      marginInlineStart: 'calc(calc(12px / 2)* -1)',
+      marginInlineEnd: 'calc(calc(12px / 2)* -1)', // causes a clipping issue
```

Comment:
the `marginInlineEnd` doesn't exist in the Spectrum solution for this. While it works, it does leave a small clipping issue so it's not ideal. The Spectrum solution leverages a second track (which would be our third), and both are positioned based on the thumb's current location.